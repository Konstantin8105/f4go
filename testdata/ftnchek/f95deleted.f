C  Tests flagging of deleted features in Fortran 95
      integer i,j,label,format
      real x
      assign 5 to label
      assign 90 to format
      j = 1
 5    continue
      do 10 x=1,5
         print format,x,j*x**2
 10   continue
      j = j+1
      if( j .eq. 2 ) goto label
      i = j*2
      assign 91 to format
      write(*,format) i,j
      write(*,91) i,j
 90   format(1x,2f6.0)
 91   format(1x,2hi=,i5,3h j=,i5)
      end

