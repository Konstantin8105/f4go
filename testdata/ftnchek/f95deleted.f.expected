COMMENT             	C  Tests flagging of deleted features in Fortran 95
NEW_LINE            	

INTEGER             	integer
IDENT               	i
,                   	,
IDENT               	j
,                   	,
IDENT               	label
,                   	,
IDENT               	format
NEW_LINE            	

IDENT               	real
IDENT               	x
NEW_LINE            	

IDENT               	assign
INT                 	5
IDENT               	to
IDENT               	label
NEW_LINE            	

IDENT               	assign
INT                 	90
IDENT               	to
IDENT               	format
NEW_LINE            	

IDENT               	j
=                   	=
INT                 	1
NEW_LINE            	

INT                 	5
continue            	continue
NEW_LINE            	

DO                  	do
INT                 	10
IDENT               	x
=                   	=
INT                 	1
,                   	,
INT                 	5
NEW_LINE            	

IDENT               	print
IDENT               	format
,                   	,
IDENT               	x
,                   	,
IDENT               	j
*                   	*
IDENT               	x
                    	**
INT                 	2
NEW_LINE            	

INT                 	10
continue            	continue
NEW_LINE            	

IDENT               	j
=                   	=
IDENT               	j
+                   	+
INT                 	1
NEW_LINE            	

if                  	if
(                   	(
IDENT               	j
==                  	.eq.
INT                 	2
)                   	)
IDENT               	goto
IDENT               	label
NEW_LINE            	

IDENT               	i
=                   	=
IDENT               	j
*                   	*
INT                 	2
NEW_LINE            	

IDENT               	assign
INT                 	91
IDENT               	to
IDENT               	format
NEW_LINE            	

IDENT               	write
(                   	(
*                   	*
,                   	,
IDENT               	format
)                   	)
IDENT               	i
,                   	,
IDENT               	j
NEW_LINE            	

IDENT               	write
(                   	(
*                   	*
,                   	,
INT                 	91
)                   	)
IDENT               	i
,                   	,
IDENT               	j
NEW_LINE            	

INT                 	90
IDENT               	format
(                   	(
INT                 	1
IDENT               	x
,                   	,
INT                 	2
IDENT               	f6
.                   	.
INT                 	91
IDENT               	format
(                   	(
INT                 	1
IDENT               	x
,                   	,
INT                 	2
IDENT               	hi
=                   	=
,                   	,
IDENT               	i5
,                   	,
INT                 	3
IDENT               	h
IDENT               	j
=                   	=
,                   	,
IDENT               	i5
)                   	)
NEW_LINE            	

END                 	end
NEW_LINE            	

NEW_LINE            	

